src/pages/_app.tsx

```diff
@@ -49,6 +49,8 @@ const USSFPortalApp = ({ Component, pageProps }: AppProps) => {
 
         <meta name="msapplication-TileColor" content="#da532c" />
         <meta name="theme-color" content="#ffffff" />
+        {/* Fix Next.js's rendering of font awesome css*/}
+        <style>{dom.css()}</style>
```

Comment:
wat

Reply:
LOL. i could not get the icon to render the correct size without this workaround, but considering you just merged in a PR with an icon that looked just fine, I need to revisit.

Reply:
I did have to add some CSS to that icon to get it to look okay though, not sure which is better

Reply:
for ref: https://github.com/FortAwesome/react-fontawesome/issues/284